doc/conf.py

```diff
@@ -88,7 +88,6 @@
     'VERSION': version,
     'WEBSITE_URL': 'https://panel.pyviz.org',
     'WEBSITE_SERVER': 'https://panel.pyviz.org',
-    'VERSION': version,
```

Comment:
How come you deleted the VERSION?

Reply:
Not related to the PR, it was just a duplicate key in that dictionary.

Reply:
Should i roll that back?

Reply:
Nevermind, I missed that. Looks fine!